spec/controllers/editorial/news_controller_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'rails_helper'
+
+RSpec.describe Editorial::NewsController, type: :controller do
```

Comment:
Really? render_views in controller specs is slowing our test suite down pretty badly I suspect ...